bin/internal/run-zowe.sh

```diff
@@ -78,6 +78,17 @@ then
   LAUNCH_COMPONENTS=${LAUNCH_COMPONENTS},files-api,jobs-api,api-mediation,explorer-jes,explorer-mvs,explorer-uss #TODO this is WIP - component ids not finalised at the moment
 fi
 
+#Explorers may be present, but have a prereq on gateway, not desktop
+if [[ $LAUNCH_COMPONENT_GROUPS == *"DESKTOP"* ]]
+then
+  LAUNCH_COMPONENTS=${LAUNCH_COMPONENTS},app-server,zss
```

Comment:
Where is components/zss created as it doesn't appear on my, or the Marist install?

Reply:
Turns out I wrote this down but didn't implement zss as a distinct component. It is within app-server. Will remove this and add a note instead.